src/items/itemDialog.component.tsx

```diff
@@ -649,6 +649,49 @@ function ItemDialog(props: ItemDialogProps) {
                                 </FormHelperText>
                               )}
                             </FormControl>
+                          ) : property.allowed_values ? (
+                            <FormControl fullWidth>
+                              <InputLabel
+                                required={property.mandatory ?? false}
+                                error={propertyErrors[index]}
+                                id={`catalogue-item-property-${property.name.replace(
+                                  /\s+/g,
+                                  '-'
+                                )}`}
```

Comment:
Could this just go into utils as a sanitise_name/remove_whitespace or something? - its used in multiple places

Reply:
#226

Reply:
Its a very small change in this case and I feel like if we come up with a case this doesn't work it would be annoying to change even though we can just put it in a common place now.

Reply:
The plan is to do each of the general change in separate prs anyways.